### ISSUE TITLE
[Priority: Medium] stdout maxBuffer exceeded

### ISSUE THREAD

**Issue by dev0tion** [stratisproject/Breeze#269]
Check logging to console when running the binaries.
As reported by @demon and @mtaipe:
```
exec error: Error: stdout maxBuffer exceeded
exec error: Error: stdout maxBuffer exceeded
```

**Comment by dev0tion** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [[Wallet-GUI] stdout maxBuffer exceeded](https://trello.com/c/OfSeSIlF/281-wallet-gui-stdout-maxbuffer-exceeded)

**Comment by dev0tion** [comment]
@nopara73 do you have an idea how to solve this?

Since we now start two daemons when running the Breeze binary we get the error above.
Has to be something related to this part of the code:
https://github.com/stratisproject/Breeze/blob/master/Breeze.UI/main.ts#L137

**Comment by nopara73** [comment]
Yes, use spawn, instead of exec, that solves the issue. However read this, just to be sure it's appropriate in Breeze:  
https://www.hacksparrow.com/difference-between-spawn-and-exec-of-node-js-child_process.html  

If you decide not to do it, then we don't really have a choice we must increase the buffer and hope for the best.

**Comment by dev0tion** [comment]
Fixed in #286 
I'm currently showing all the output from both daemons when you start the binary in command line.
I've added Bitcoin for Bitcoin logging and Stratis for Stratis logging.

This should help for debugging.
